src/litdata/streaming/reader.py

```diff
@@ -12,12 +12,11 @@
 # limitations under the License.
 
 import contextlib
-import multiprocessing
 import os
 import warnings
 from logging import Logger
-from queue import Empty
-from threading import Thread
+from queue import Empty, Queue
```

Comment:
The `queue.Queue` is suitable for multi-thread, and `multiprocessing.Queue` may cause unexpected behavior in multi-thread setting.